### DISCUSSION TITLE
Do we need to update the readme?

### DISCUSSION THREAD

**Discussion by beckynevin** [deepskies/DeepBench#83] [General]
We should probably at least add information about the pendulum to the readme so that when we merge with main it will be included.

I'm wondering if there are other parts people would like to update.

It looks like the deepbench logo isn't working?

**Comment by voetberg** [comment]
Yeah we absolutely need to update that before we make the code public. I think rn, the main readme is maybe? 2 years out of date?

**Reply by bnord** [reply]
the main readme is waaaaay out of date.

Nothing there is precious. 

Should we include a contribution statement/instructions for folks who might want to build on it?

**Reply by voetberg** [reply]
Contributor guidelines are always a good idea, (though I admit I've never written them form scratch so I will defer to Brian on what he'd like to see)

**Reply by bnord** [reply]
I like this one (but it was also the first time I started paying attention to contributor guidelines.)
https://github.com/skypyproject/skypy/blob/main/CONTRIBUTING.rst

**Reply by voetberg** [reply]
Oh this is WAY more descriptive than I would have written. What's their license, can we just steal this?

**Reply by bnord** [reply]
I'm a part of the collaboration, and a long time ago they said we're welcome to pull from it. I think we should just give them credit somewhere.
